### ISSUE TITLE
Recreate batch function

### ISSUE THREAD

**Issue by jobbl** [jobbl/KP_MedInf#1]
recreate their batch function in pytorch

**Comment by jobbl** [comment]
apply dropout layer in good place

**Comment by jobbl** [comment]
play around with hyperparameters, maybe make hyperparameter search
